tests/cpp/operator/mkldnn_operator_test.cc

```diff
@@ -866,6 +925,12 @@ TEST(IMPERATIVE, LRNOp) {
   TestOpEx(forward_attrs, backwards_attrs);
 }
 
+TEST(IMPERATIVE, SoftmaxOp) {
+  OpAttrs forward_attrs = GetSoftmaxOp();
+  OpAttrs backwards_attrs;
```

Comment:
Why not have a function GetSoftmaxBackwardsOp() like in line https://github.com/apache/incubator-mxnet/pull/12884/files#diff-472f3cc48fa2159cf280bf25b15417eaR936

Reply:
MKLDNN implementation not present for backward softmax operator yet.